src/Sylius/Bundle/WebBundle/Resources/views/Backend/Locale/macros.html.twig

```diff
@@ -8,24 +8,27 @@
     <table class="table">
         <thead>
             <tr>
+                <th>{{ sylius_resource_sort('id', 'sylius.locale.id'|trans) }}</th>
+                <th>{{ 'sylius.locale.name'|trans }}</th>
                 <th>{{ sylius_resource_sort('code', 'sylius.locale.code'|trans) }}</th>
                 <th>{{ sylius_resource_sort('enabled', 'sylius.locale.enabled'|trans) }}</th>
-                <th>{{ sylius_resource_sort('updatedAt', 'sylius.locale.updated_at'|trans) }}</th>
                 <th></th>
             </tr>
         </thead>
         <tbody>
         {% for locale in locales %}
-            <tr id="{{ locale.id }}">
+            <tr>
+                <td>{{ locale.id }}</td>
```

Comment:
We have unique code, so no need for ID. IMO the table should be the following:

| Code        | Name           | Enabled  | Buttons |
| ------------- |:-------------:| -----:| -------:|